support/Pipelines/Bbh/Inspiral.yaml

```diff
@@ -111,7 +117,7 @@ Evolution:
         SafetyFactor: 0.95
   TimeStepper:
     AdamsBashforth:
-      Order: 5
+      Order: 4
```

Comment:
Also add a comment here. Basically, please add a comment for everything you changed for a reason, so we remember the reason :)